### ISSUE TITLE
Make the UI faster for frequent users

### ISSUE THREAD

**Issue by dickolsson** [MaKleSoft/padlock#174]
I interact with the Padlock app probably 1000 times as I go through my day, via various actions; lock/unlock/search/copy/paste etc.

There are two things that keep bugging me in terms of the user experience.

## 1. Animations

The animations are great, and I love the look and feel of the app.
But those fade/slide animations of 0.5 sec (or whatever it is) every time I unlock the app feels a bit redundant, or too slow. I know I can already start typing on the keyboard to search for my record as the animation is happening, it's just not as fast I'd like.

One other animation that could be faster is the one when the right pane is closed/blacked out, and I click on one of my records to scroll through the entries (it opens like a card from the side) - this animation could be a bit faster.

Bottom line: I'd like the animations to be faster, or perhaps have an option to turn them off completely.

## 2. Tabbing

There's one situation that occur when the macOS desktop app has gone out of focus and automatically locked itself - when I bring up Padlock again and try to move the cursor focus to the "Enter Master Password" field by hitting the `tab` key it does not work.

In fact, if I do hit tab I'll notice that nothing was focused. And upon login the UI will be broken:

<img width="797" alt="screen shot 2018-04-23 at 12 36 32" src="https://user-images.githubusercontent.com/112085/39124389-2d02e36e-46f3-11e8-9f97-93af2febefc7.png">

Bottom line: I'd like to be able to navigate the UI entirely by using the keyboard.

Thanks for such a great app!

**Comment by dickolsson** [comment]
Just noticed that there's both #151 and #173 for the tabbing/keyboard navigation bit. So I guess the scope of this ticket can be focused on the animations bit only.

**Comment by dickolsson** [comment]
@MaKleSoft Thanks for the quick response. I trimmed down this issue to just be focused on the animations bit. I'll open up a new issue for the UI bug with tabs

**Comment by dickolsson** [comment]
I added #175 for the tabbing bug.
